docs/pwa/pwa.md

```diff
@@ -0,0 +1,67 @@
+# Progressive Web App tools
```

Comment:
These docs so far are a useful guide to what the service worker is doing - it might be useful to read this or keep it open when looking at the service worker code